### ISSUE TITLE
Description for series script template not shown when apparently it's supposed to be taken from the first episode if empty

### ISSUE THREAD

**Issue by scr0llbaer9000** [TVTower/TVTower#924]
For this script, no description is shown.

![image](https://user-images.githubusercontent.com/123826268/221748165-8dcfb11e-abc9-4c09-97fe-7bafa753c7c6.png)

According to the comment, it is supposed to be taken from the first episode.

		<scripttemplate product="2" licence_type="3" guid="scripttemplate-ron-visiting">
			<title>
				<de>${ttl}</de>
				<en>${ttl}</en>
			</title>
			<description>
				
				<de></de>
				<en></en>
			</description>
			<variables>
				<ttl>
					<de>Zu Gast in ${cntry}|Reise durch ${cntry}|Auf Achse in ${cntry}|Leben in ${cntry}|Sehenswertes ${cntry}|Faszination ${cntry}|Blickpunkt ${cntry}</de>
					<en>Visiting ${cntry}|Travelling in ${cntry}|On the road in ${cntry}|Life in ${cntry}|Beautiful ${cntry}|Fascinating ${cntry}|Spotlight ${cntry}</en>
				</ttl>
				<cntry>
					<de>Deutschland|Spanien|Frankreich|Dänemark|Schweden|Finnland|Norwegen|Polen|Griechenland|Italien|Portugal|Brasilien|Costa Rica|Kanada|Vietnam|Thailand|Japan|Belgien|Nepal|Inden|Chile</de>
					<en>Germany|Spain|France|Denmark|Sweden|Finland|Norway|Poland|Greece|Italy|Portugal|Brazil|Costa Rica|Canada|Vietnam|Thailand|Japan|Belgium|Nepal|India|Chile</en>
				</cntry>
			</variables>
			
			<children>
				<scripttemplate product="2" licence_type="2" index="0" guid="scripttemplate-ron-visiting-ep1">
					
					<title>
						<de>${cntry} im Überblick</de>
						<en>${cntry} in a nutshell</en>
					</title>
					<description>
						<de>Geografische, historische und politische Zusammenfassung zu ${cntry}, bevor sich die Folgen einzelnen Aspekten des Landes widmen.</de>
						<en>Geographical, historical and political summary of ${cntry}, before a specific aspect of the country is highlighted in each episode.</en>
					</description>
					<outcome min="30" max="60" slope="50" />
				</scripttemplate>

**Comment by nittka** [comment]
The comment is almost correct (or correct if read differently). If the parent has no description tag at all, the description of the first episode would be used. However, when removing it, the first episode's description would be used for all episodes which is not intended either.

The meaning of the comment is: an empty description is set here because otherwise everything in the series gets the description of the first episode.

**Comment by scr0llbaer9000** [comment]
you sure that's what @GWRon intended? Still reads differently to me, and it also would make sense, because the description of the first episode is very generic.

**Comment by GWRon** [comment]
https://github.com/TVTower/TVTower/commit/cd7fec4dc47ea438cca0d326fa26188cf27d445d

Think they are Alex' data.

**Comment by nittka** [comment]
Yes, my data. I was too "lazy" to create a user file for those three scripts...
The description of the first episode is generic, but is states "summary before specific aspects ... in each episode". It really was intended as  description for just the first episode. And there is a TODO for topic specific descriptions.
